sermant-plugins/sermant-router/spring-router-plugin/src/main/java/io/sermant/router/spring/interceptor/DiscoveryManagerInterceptor.java

```diff
@@ -59,6 +59,7 @@ public ExecuteContext before(ExecuteContext context) {
             Object serviceName = ReflectUtils.getFieldValue(obj, "serviceName").orElse(null);
             if (serviceName instanceof String) {
                 AppCache.INSTANCE.setAppName((String) serviceName);
+                configService.init(RouterConstant.SPRING_CACHE_NAME, AppCache.INSTANCE.getAppName());
```

Comment:
Directly use servicename instead of AppCache.INSTANCE.getAppName()

Reply:
in spring framework, servicename is a stirng